relationships/candidates/AWSDYNAMODBTABLE.yml

```diff
@@ -12,6 +12,8 @@ lookups:
           field: awsRegion
         - tagKeys: ["aws.tableName"]
           field: awsTableName
+        - tagKeys: ["aws.arn"]
```

Comment:
This is redundant and should be move to another rule.
Entity should match awsAccountId AND awsRegion AND awsTableName, OR arn.

Reply:
Can you please suggest that how I should change.?

Reply:
You could add a different `lookup` rule for ARN as we do in the candidate definition for `HOST`: https://github.com/newrelic/entity-definitions/blob/main/relationships/candidates/HOST.yml

Reply:
I've just added aws.arn and other attributes are removed. Can please have a look.

Reply:
is intended that account, region & table are nowhere in any rule now?

Reply:
arn contains all that, right? [LINK](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html)
```
arn:partition:service:region:account-id:resource-id
arn:partition:service:region:account-id:resource-type/resource-id
arn:partition:service:region:account-id:resource-type:resource-id
``` 
so what I understand is that this rule is for ARN only and there will be another one for accountId+Region+TableName. Is this correct @DasuNewrelic ?

Reply:
Yes, arn contains all.